interfaces/HO-Portal/src/app/programs/program-details/program-details.component.html

```diff
@@ -4,14 +4,25 @@
       <ion-menu-button></ion-menu-button>
     </ion-buttons>
     <ion-title>
-      {{ 'page.programs.PROGRAM_DETAILS'|translate }}
+      <ion-icon name="arrow-round-back" [routerLink]="['/programs']"></ion-icon>
```

Comment:
you already changed this right?

Reply:
Yeah, 80%.

But it turns out it isn't fully doing what I thought it would do... So I've refatored it into a 'proper' button into its 'proper' place....